Client/src/Common/Submitter/ClientServiceConnector.cs

```diff
@@ -38,7 +38,7 @@ public class ClientServiceConnector
 #if NET5_0_OR_GREATER
       var doOverride = !string.IsNullOrEmpty(options.CaCert);
 #else
-      var doOverride = true;
+      var doOverride = properties.ControlPlaneUri.Scheme.Contains("https");
```

Comment:
I think this does not work on Windows 10. So there should most likely be a test on the OS. You could ask @dbrasseur-aneo to be sure.

Reply:
I have issues to test due to strong name limitations (On Utils and LanguageExt).
I would suggest going with adding the condition "properties.ControlPlaneUri.Scheme == Uri.UriSchemeHttps && " to the if above, instead of in the doOverride.